### ISSUE TITLE
[Feature Request]: Ability to export raw chat logs for model troubleshooting.

### ISSUE THREAD

**Issue by Stevie-O** [google-ai-edge/gallery#1067]
**Is your feature request related to a problem? Please describe.**    
Not a problem with AI Edge Gallery itself, but a problem I experienced with one of the models.  Actually, a *lot* of problems I experience with the main model I have (Gemma 4 E2B).

I had a **bizarre** transformer failure that caused the model to output nonsense for several tokens before restabilizing.  From the app, all I wanted was something simple: To take the entire conversation and bring it over to my PC with its original contents, unaltered.  I could then spin up Pytorch and TransformerLens see what the logits look like.

Well, apparently the chat files are buried deep inside my phone where _I cannot seem to access them, even with adb_.

I'm even okay with the raw protobuf data!  That would be preferable, actually, because then I could have confidence that the thinking tokens were preserved and no stray whitespace tokens were added.

**Describe the solution you'd like**
I would like a way to go to the app and either directly export a conversation from the chat history, or be able to export the _currently loaded_ conversation from the chat history, into a place that I can then copy it to my computer.

**Describe alternatives you've considered**
I tried using ADB to extract the files and was rebuffed.
I am not interested in building a custom version of this app with debugging enabled just to do this one thing.

**Additional context**
(none)

**Comment by dpknag** [comment]
Hi @Stevie-O,

Thank you for opening this feature request and sharing your detailed use case.

We have noted this feedback regarding raw chat log and protobuf exports, and along with other related export requests. We will keep tracking this request and thank you for taking time to report this.

Thank you!
